src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -62,6 +49,18 @@ protected void releaseBrowser() {
     }
 
     /**
+     * Reminder to disable GodMode and re-run the test(s).
+     */
+    @AfterSuite
+    public static void remindUserToDisableGodModeIfRequired() {
+        if (TestProperties.IS_GODMODE_ENABLED) {
```

Comment:
This would imply that we don't need to remind the user to disable GodMode if it's switched on in a CI environment, which doesn't sound right to me.